core/src/test/java/com/google/errorprone/bugpatterns/flogger/FloggerWithCauseTest.java

```diff
@@ -105,7 +105,7 @@ public void method(Exception e) {
         .doTest();
   }
 
-  // regression test for http://b/29131466
+  // regression test for https://b/29131466
```

Comment:
All of the `b/` links should stay as `http:`

Reply:
I've reverted `/b/` links